### DISCUSSION TITLE
Inference onnx converted Topdown HRNET model

### DISCUSSION THREAD

**Discussion by QuentinADH** [open-mmlab/mmdeploy#2539] [Q&A]
Hi everyone, 

After converting td-hm_hrnet-w48_udp-8xb32-210e_coco-384x288 model into onnx. I would like to inquire about the inference.
I succeed to apply the inference as referred to the following URL, however this code is only usable for one detected bounding box.
https://mmdeploy.readthedocs.io/en/stable/04-supported-codebases/mmpose.html#model-inference

Do you have by any chance any example I could take as reference that would deal with several bounding box in the image?

Thank you for your time.

**Comment by me7obeast** [comment]
Excuse me, but did you find any solution yet? I ran into the exact same problem. I think a detector is needed here ( top-down method ), but I dunno how to pass the results of a detector to an onnx pose model
